### PR TITLE
 feat: add footer component

### PR DESCRIPTION
**Pull request by AdelineA**
- add footer component

### CODE REVIEW

File: src/components/footer/data-source.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import output from "../../../public/data.json";
+const dataSources = output
+  .map((data) => data?.PARSED_MANUAL_TAGS?.ACCESS)
+  .flat()
+  .filter((source) => source && !source.includes("Open Access"))
+  .filter((source) => source) as string[];
+
+export const uniqueDataSources = Array.from(new Set(dataSources));
```
Comment (NNesta) [inline src/components/footer/data-source.ts:8]: Can you change this name of this variable to name that give hint what it hold

File: src/components/footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <footer className="text-center my-4 p-4 px-8 text-base font-bold float-end border-2 border-gray-600 mr-14">
+      <h2 className="mb-4">All Resources are Open Access: unless indicated:</h2>
+      {uniqueDataSources.map((source, index) => {
+        const color = getColorByAccess(source);
+        return (
+          <div
+            key={index}
+            className="flex gap-5 w-full justify-center items-center"
+          >
```
Comment (NNesta) [inline src/components/footer/index.tsx:16]: Can you align the content to the right side as on the design

**Review by NNesta — CHANGES_REQUESTED** [review]

**Review by NNesta — APPROVED** [review]
LGTM

File: src/components/footer/data-resources.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import output from "../../../public/data.json";
+
+const dataResources = output
```
Comment (musayann) [inline src/components/footer/data-resources.ts:3]: these should be in the `src/data` folder

**Review by musayann — CHANGES_REQUESTED** [review]

File: src/data/footer/index.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Star from "../../components/icons/Star";
+import { getColorByAccess } from "../../utils/helpers";
+import { filteredResources } from "./data-resources";
+
+const Footer = () => {
```
Comment (NNesta) [inline src/data/footer/index.tsx:6]: Can you put this component inside components folder

**Review by NNesta — CHANGES_REQUESTED** [review]

**Review by NNesta — APPROVED** [review]
LGTM

**Review by musayann — APPROVED** [review]

**Pull request merged**